tpot/base.py

```diff
@@ -1059,9 +1066,14 @@ def _evaluate_individuals(self, individuals, features, target, sample_weight=Non
                 for val in tmp_result_scores:
                     result_score_list = self._update_val(val, result_score_list)
 
-        self._update_evaluated_individuals_(result_score_list, eval_individuals_str, operator_counts)
+        self._update_evaluated_individuals_(result_score_list, eval_individuals_str, operator_counts, stats_dicts)
+
+        def get_optimization_values(individual):
```

Comment:
`get_optimization_values` should be an internal function to TPOT, no? Otherwise it's constantly being re-defined every time `_evaluate_individuals` is being called.

Reply:
Actually, is this function needed at all? It's only used once, below.

Reply:
The function is indeed only used once below, but i find the list compresension in which the function is called better readable in this way. Will change it